[TH]_Psycho_Pass_THE_MOVIE.ass

```diff
@@ -884,48 +884,48 @@ Dialogue: 0,1:19:03.82,1:19:05.95,Default,,0,0,0,,are being measured accurately.
-Dialogue: 0,1:22:33.86,1:22:37.32,Default,,0,0,0,,I was under the impression that the whole country had been turned into
-Dialogue: 0,1:22:37.41,1:22:39.95,Default,,0,0,0,,spineless, phony wimps by the Sybil System.
-Dialogue: 0,1:22:42.08,1:22:45.67,Default,,0,0,0,,Well, it's true that I don't have a place there anymore.
-Dialogue: 0,1:22:49.59,1:22:52.76,Default,,0,0,0,,Did you decide to quit government work after getting professional training?
-Dialogue: 0,1:22:52.84,1:22:55.51,Default,,0,0,0,,But you're still only a third rate mercenary.
-Dialogue: 0,1:22:55.63,1:22:59.35,Default,,0,0,0,,After all, you still haven't grasped how to choose the right employer.
+Dialogue: 0,1:19:21.13,1:19:22.92,Default,,0,0,0,,อ๊ะ แล้วก็อีกเรื่องนึง
+Dialogue: 0,1:19:23.05,1:19:27.13,Default,,0,0,0,,ปลอกคอที่เอามาใช้ควบคุมค่าอาชญากรแฝงบนเกาะชัมบาล่านี่
+Dialogue: 0,1:19:27.30,1:19:29.68,Default,,0,0,0,,ถูกทำขึ้นที่ญี่ปุ่นสินะคะ ?
+Dialogue: 0,1:19:30.47,1:19:32.97,Default,,0,0,0,,คุณคิดว่าคุณสามารถใช้ชื่อของหน่วยรักษาความสงบ เพื่อเอารหัสปลดล๊อก
+Dialogue: 0,1:19:33.14,1:19:34.60,Default,,0,0,0,,ให้ฉันได้รึเปล่าคะ ?
+Dialogue: 0,1:19:34.68,1:19:37.85,Default,,0,0,0,,ก็ดูง่ายดีนะ แต่เธอจะไม่เป็นไรจริงเหรอ ?
+Dialogue: 0,1:19:37.98,1:19:39.65,Default,,0,0,0,,ฉันไม่คิดว่าพวกทหารรักษาการณ์จะปลื้มหรอกนะ
+Dialogue: 0,1:19:40.44,1:19:43.11,Default,,0,0,0,,ตอนนี้มันเลยจุด
+Dialogue: 0,1:19:43.28,1:19:45.61,Default,,0,0,0,,ที่จะทำให้คนอื่นพอใจแล้วค่ะ
+Dialogue: 0,1:19:47.15,1:19:49.78,Default,,0,0,0,,อาหารพร้อมแล้วค่ะ
+Dialogue: 0,1:19:52.78,1:19:53.99,Default,,0,0,0,,ขอบคุณนะ
+Dialogue: 0,1:20:03.00,1:20:04.05,Default,,0,0,0,,อยู่เงียบๆ นะ
+Dialogue: 0,1:20:10.26,1:20:11.85,Default,,0,0,0,,น..นี่มัน
+Dialogue: 0,1:20:12.68,1:20:15.27,Default,,0,0,0,,ใจเย็นๆ แล้วฟังที่ฉันพูดก่อนนะ
+Dialogue: 0,1:20:15.35,1:20:17.35,Default,,0,0,0,,ฉันต้องการให้เธอร่วม...
+Dialogue: 0,1:20:18.48,1:20:19.35,Default,,0,0,0,,มือ...
+Dialogue: 0,1:20:29.11,1:20:30.74,Default,,0,0,0,,นี่มัน...
+Dialogue: 0,1:20:50.72,1:20:52.39,Default,,0,0,0,,ฉันทำตามที่คุณขอแล้ว
+Dialogue: 0,1:20:52.51,1:20:56.43,Default,,0,0,0,,คุณจะปลดปลอกคอให้พี่ชายของฉันตอนนี้เลย ได้ไหม ?
+Dialogue: 0,1:20:56.64,1:20:58.68,Default,,0,0,0,,ฉันจะเอากลับไปคิดดีๆ ก็แล้วกัน
+Dialogue: 0,1:21:08.57,1:21:10.28,Default,,0,0,0,,อย่าเพิ่งฆ่านังนี่ล่ะ
+Dialogue: 0,1:21:10.40,1:21:14.24,Default,,0,0,0,,ฉันจะจัดฉากให้เธอตายไปพร้อมกับพวกกลุ่มโจร
+Dialogue: 0,1:21:14.37,1:21:17.79,Default,,0,0,0,,จะได้ไม่มีใครสงสัย
+Dialogue: 0,1:21:18.95,1:21:21.37,Default,,0,0,0,,ฉันรู้ว่านะ
+Dialogue: 0,1:21:21.87,1:21:24.54,Default,,0,0,0,,ไซแมนติกสแกนน่ะ
+Dialogue: 0,1:21:25.46,1:21:27.50,Default,,0,0,0,,พวกแกหลบเลี่ยงมันได้
+Dialogue: 0,1:21:29.30,1:21:30.59,Default,,0,0,0,,โอ้
+Dialogue: 0,1:21:30.80,1:21:32.59,Default,,0,0,0,,เธอมีหลักฐานรึไง
+Dialogue: 0,1:21:33.30,1:21:36.35,Default,,0,0,0,,ให้ตายเถอะ ! คุณเพิ่งฆ่าคนไป
```

Comment:
ของ eng มันแปลแล้วมีคำว่า  for god's sakeมาด้วยอ่ะ มันเป็นอารมณ์ประมาณว่าอุทานขึ้นมาอ่ะ